lib/srv/db/access_test.go

```diff
@@ -1276,6 +1279,9 @@ type testContext struct {
 	opensearch map[string]testOpenSearch
 	// dynamodb is a collection of DynamoDB databases the test uses.
 	dynamodb map[string]testDynamoDB
+	// clickHouse is a collection of ClickHouse databases the test uses.
+	clickHouse map[string]testClickHouse
```

Comment:
Right now the ClickHouse DB engine is multiplexing HTTP and Native protocol so I'm not seeing the point of splitting the `clickHouse map[string]testClickHouse` per ClickHouse protocol since in this implementation we are doing the same in  ClickHouse protocol in `db/clickhosue/engine.go` where we are handling multiple protocol in one db ClickHouse engine.

Reply:
Sure, in the context of this particular PR this makes perfect sense. I mean this in the larger sense of all other protocols.